docs/README.md

```diff
@@ -1,4 +1,4 @@
-# Why Helm?
+# Helm 101
```

Comment:
Would have been nice to keep the old title .. less formal?

Reply:
same as above

Reply:
Replied above.